mytools.py

```diff
@@ -740,3 +751,21 @@ def intersection(boxA, boxB):
 
     # compute the area of intersection rectangle
     return dx * dy
+
+
+def predict_stream(model, camera_id):
```

Comment:
Is this a temporary solution to show cleaner code while we add the predict_stream method to model class?

Reply:
Well, it depends. To implement this in PySDK we will need to add new dependency to PySDK:
`pafy`, which is LGPLv3. Do we want to?

Reply:
Tough luck. I guess we need to keep it outside. But can we now include LDPLv3 inside MIT (pysdkexamples)?
I want to at least change the name to dgtools from  mytools (at least the parts that are well tested and mature)